data/json/recipes/basecamps/recipe_modular_field_tent.json

```diff
@@ -86,7 +86,7 @@
   },
   {
     "type": "recipe",
-    "result": "faction_base_modular_hub_tent_northwest",
+    "result": "faction_base_modular_hub_tent_southwest",
     "description": "We should expand our housing by putting up a tent on the southwest side, though doing so will mean we will need more materials to build the central building.",
```

Comment:
```suggestion
    "description": "We should expand our housing by putting up a tent on the southwest side, though doing so will mean we need more materials to build the central building.",
```

Reply:
I'm not sure where the space before `"description"` came from, but me edit didn't touch that part of the line, it just removed the word "will".

Reply:
There are many other descriptions that have the redundant "will". Did you want me to fix all of them (for consistency)?

Reply:
Technically not your responsibility, but I think it would fit in this PR. If you choose to fix now, it would be much appreciated.

Reply:
I don't mind doing it, it's just that I've never used Git before and I can't figure out how to get the commit I just did via the github UI back into my local repository. I'm using egit on eclipse on Windows to make matters more complicated. I'll see what I can do.

Reply:
Ok, done.

Reply:
As counter-intuitive as it may be, using the 'built-in git editor' is highly discouraged. It doesn't follow/adhere to the same json styling this project requires/demands, which leads to plenty of confusion when various checks fail. While I'm useless as far egit is concerned, [this guide](https://github.com/CleverRaven/Cataclysm-DDA/wiki/Guide-to-adding-new-content-to-CDDA-for-first-time-modders:-Getting-Started) may be a good starting point. If you're still not confident, leave this PR as is, and someone else will take up the corrections.

Reply:
That was written before I saw your last commit, so it seems my advice is unneeded.

Reply:
I've already made the corrections (unless there is something I missed or misunderstood). I don't think I have any style violations (let me know if I do).

Reply:
I think you're good, sorry for the hassle :P